internal/provider/provider.go

```diff
@@ -70,6 +70,7 @@ func New() *schema.Provider {
 			"boundary_auth_method_oidc":         resourceAuthMethodOidc(),
 			"boundary_credential_library_vault": resourceCredentialLibraryVault(),
 			"boundary_credential_store_vault":   resourceCredentialStoreVault(),
+			"boundary_credential_store_static":  resourceStaticCredentialStore(),
```

Comment:
Nit, can we name the function similarly to the others too?
```suggestion
			"boundary_credential_store_static":  resourceCredentialStoreStatic(),
```